### PR TITLE
Allow For Shape To Be Updated

### PR DESCRIPTION
**Pull request by eudisd**
`update-embed` is an event I post from the dashboard-embed project, with all the necessary data.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -601,6 +601,14 @@ export default function Vidy(opts) {
 		}
 	};
 
+	$.updateShape = (node, shape) => {
```
Comment (lukeed) [inline src/index.js]: Does not go here

File: src/edits.js [hunk 2]
```diff
@@ -77,5 +77,10 @@ window.addEventListener('message', e => {
 	} else if (task === 'refresh') {
 		location.reload();
 		ctx.emit('init', { appid: ctx.appid, postid: ctx.postid });
+	} else if (task === 'update-embed') {
+		const id = value.id;
+		const shape = value.shape;
+		const node = document.querySelector(`#${id}`);
+		ctx.updateShape(node, shape);
```
Comment (lukeed) [inline src/edits.js]: You might as well write in base64 😆 

```js
const elem = document.getElementById(value.id);
ctx.setShape(elem, value.shape);
```

File: src/index.js (hunk 1 above)
Comment (eudisd) [reply src/index.js]: Where do you want me to put this?  I need access to `video` and `ctx` does not expose it.

File: src/edits.js (hunk 2 above)
Comment (eudisd) [reply src/edits.js]: 🤘

**Comment by eudisd** [comment]
Gotcha thanks @lukeed, I'll try out the way you outlined.   Also rebased and changed base branch of PR. Will keep you posted

**Comment by eudisd** [comment]
Oh and I'll double check if the event updates the player while it's open

**Comment by eudisd** [comment]
@lukeed I pushed up the requested changes.  Let me know if there is anything else

**Comment by eudisd** [comment]
Hey @lukeed so if I have the sdk opened on a different tab, with the player opened (in full screen) and then update the shape in the dashboard (via a publish), it does not update the shape of the player real time in the sdk when it's opened.  It also does not update the shape of the regular small (on hover) player when you hover over a demarcation.  

If I reload the sdk page then it picks up the change

**Comment by eudisd** [comment]
So we're good to go here I think

**Pull request merged**